internal/services/automanage/migration/configuration_v0_to_v1.go

```diff
@@ -0,0 +1,338 @@
+package migration
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/go-azure-sdk/resource-manager/automanage/2022-05-04/configurationprofiles"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+var _ pluginsdk.StateUpgrade = ConfigurationV0ToV1{}
+
+type ConfigurationV0ToV1 struct {
+}
+
+func (ConfigurationV0ToV1) Schema() map[string]*pluginsdk.Schema {
```

Comment:
Per our [state migration guide](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/guide-state-migrations.md) we shouldn't need to have `ForceNew`, `Default` and `MaxItems` in the schema for state migrations?